flags.go

```diff
@@ -309,6 +309,12 @@ func newApp() (app *cli.App) {
 					"When this flag is specified, GCSFuse uses the GCS JSON API instead.",
 			},
 
+			cli.BoolTFlag{
+				Name: "enable-managed-folders",
```

Comment:
can we rename this flag to enable-managed-folders-listing?

As `enable-managed-folders`  feels like we are controlling the managed folder behavior which is not true.